build-aux/git-version-gen is GPLv3 licensed, so remove it.

Upstream-Status: Inappropriate [License Issue]

Signed-off-by: Kai Kang <kai.kang@windriver.com>
---
--- corosync-2.3.4/configure.ac.orig	2014-12-26 16:51:32.052522492 +0800
+++ corosync-2.3.4/configure.ac	2014-12-26 16:55:29.364528960 +0800
@@ -5,7 +5,7 @@
 AC_PREREQ([2.61])
 
 AC_INIT([corosync],
-	m4_esyscmd([build-aux/git-version-gen .tarball-version]),
+	2.3.4,
 	[discuss@lists.corosync.org])
 
 AC_USE_SYSTEM_EXTENSIONS
